test/remote/gateways/remote_adyen_test.rb

```diff
@@ -1015,11 +1015,9 @@ def test_successful_store_with_elo_card
     assert_equal 'Authorised', response.message
   end
 
-  # Adyen does not currently support recurring transactions with Cabal cards
   def test_failed_store_with_cabal_card
```

Comment:
💬 Comment:

My understanding of this change is that Adyen now supports cabal cc, if that is the case the test title is wrong because you are asserting a success.